std_msgs/Header header
geometry_msgs/Pose[] ee_poses
geometry_msgs/Twist[] tolerances
